src/components/vm/consoles/consoles.jsx

```diff
@@ -133,27 +163,40 @@ export const ConsoleCard = ({ state, vm, config, onAddErrorNotification, isExpan
                         inactiveConsoleDetail={inactive_vnc}
                         spiceDetail={spice}
                         onAddErrorNotification={onAddErrorNotification}
-                        isExpanded={isExpanded} />
+                        isExpanded={isExpanded || isStandalone}
+                        onRemoteSizeChanged={(w, h, mode) => {
+                            lastVncRemoteSize.current = [w, h];
+                            if (isStandalone && mode == "none") {
+                                let header_height = 53;
+                                const title = document.querySelector(`#${vmId(vm.name)}-consoles .pf-v6-c-card__header`);
+                                if (title)
+                                    header_height = title.offsetHeight;
+                                const delta_width = window.outerWidth - window.innerWidth;
+                                const delta_height = window.outerHeight - window.innerHeight;
+                                window.resizeTo(w + delta_width, h + header_height + delta_height);
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2254-839cfa62-20250805-072522-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)